js/script.js

```diff
@@ -423,3 +423,31 @@ contactForm.addEventListener('submit', (e) => {
     error[2].parentElement.classList.add('success');
   }
 });
+
+// SAVE TO LOCAL STORAGE
+const clientName = document.getElementById('name');
+const clientEmail = document.getElementById('email');
+const clientMessage = document.getElementById('message');
+
+function useLocalStorage() {
+  const contactMessage = {
+    name: clientName.value,
+    email: clientEmail.value,
+    mail: clientMessage.value,
```

Comment:
### [OPTIONAL]
- [ ] kindly use the appropriated name for the message input, you can use `message` instead of `mail`.